### ISSUE TITLE
GUC - Economy is bound to get nuked

### ISSUE THREAD

**Issue by MrStraws114514** [TNODevs/TNO-Bug-Tracker#3375]
### Operating System

Windows 11

### HOI4 version

1.14.7

### TNO version

1.5.0d

### Do you have other mods in use?

None

### Were you using Steam?

No

### Which major expansions do you NOT have?

I have all

### Describe the bug

GUC is too poor, the whole country only has 1 state, and this state only has 0.01B GDP, VAC and MOC have the same situation
however, different with them, GUC has an unsourced military expenditure, even if it has no any unit, it still need to cost 0.002B for it

And this is the biggest problem, 0.002B is enough to make GUC's debt out of control
GUC's debt-to-GDP will rise 20% every month, that means it only needs a year to get 200% debt-to-GDP
Coupled with non-content countries cannot fire Economic crises to save the debt, economic deficits and debt interest rates will doom this tag economy to get nuked

### Steps to reproduce / Cause of the issue

Idk where GUC's military spending comes from, but removing it would solve the problem
If can't be removed, then this can only be solved by increasing population and GDP, or add a hidden income buff

### Expected behaviour

GUC's economy will be sustainable, not constantly burdened with debt

**Comment by Marok132** [comment]
The TNO economy often gets nuked in some countries (West African nations in particular). I should say TNO devs have to do something with it...

**Comment by MrStraws114514** [comment]
> The TNO economy often gets nuked in some countries (West African nations in particular). I should say TNO devs have to do something with it...

yes, when a country get too many debt-to-rate, its GDP growth will get very low, then sometimes it will even overflow to a very high number, make a state very rich, this is common in Kemerovo

but debt reduce GDP growth is working as designed, everything are working correctly but get a wrong ending, its impossible to completely fixed, just like minecraft far lands
fiscal crisis system also has a special solution to reduce AI's debt, but it needs 6 months to fire, too late for those countries in war

I think it is possible to check the AI's debt every month, and when it is too high, immediately trigger an event to suppress the debt to a reasonable value, but just wait for someone who wanna do it..
